modules/040-control-plane-manager/images/control-plane-manager/controller/kubeconfig.go

```diff
@@ -36,7 +37,14 @@ import (
 
 func renewKubeconfigs() error {
 	log.Info("phase: renew kubeconfigs")
-	for _, v := range []string{"admin", "controller-manager", "scheduler"} {
+
+	kubeconfigs := []string{"admin", "controller-manager", "scheduler"}
+	// if KubernetesVersion >= v1.29.0
+	if semver.Compare("v1.29.0", fmt.Sprintf("v%s", config.KubernetesVersion)) < 1 {
+		kubeconfigs = []string{"super-admin", "admin", "controller-manager", "scheduler"}
```

Comment:
can we symlink kubectl config to super-admin config ?

Reply:
We can. It works.

```shell
root@dev-master-0:~# ls -l .kube/
total 4
drwxr-x--- 4 root root 4096 Mar 26 21:02 cache
lrwxrwxrwx 1 root root   32 Mar 28 14:36 config -> /etc/kubernetes/super-admin.conf
root@dev-master-0:~# kubectl get node
NAME                              STATUS   ROLES                  AGE   VERSION
dev-master-0                      Ready    control-plane,master   41h   v1.29.3
dev-worker-7badd9ba-5ddbc-9lx9c   Ready    worker                 41h   v1.29.3
```